include/task.h

```diff
@@ -1795,8 +1795,10 @@ UBaseType_t uxTaskGetSystemState( TaskStatus_t * const pxTaskStatusArray,
  * demo applications.  Do not consider it to be part of the scheduler.
  *
  * vTaskList() calls uxTaskGetSystemState(), then formats part of the
- * uxTaskGetSystemState() output into a human readable table that displays task
- * names, states and stack usage.
+ * uxTaskGetSystemState() output into a human readable table that displays task:
+ * names, states, priority, stack usage and task number.
+ * Stack usage specified as the number of unused variables stack can hold
```

Comment:
This is very imprecise as variables can be almost any number of bytes.  Does it mean words?  A word being 16-bits on 16-bit devices, 32-bits on 32-bit devices?

Reply:
Replaced with: Stack usage specified as the number of unused StackType_t words